packages/functions/src/api/service.ts

```diff
@@ -71,7 +71,7 @@ export class Service implements FirebaseFunctions, FirebaseService {
   private readonly serializer = new Serializer();
   private emulatorOrigin: string | null = null;
   private cancelAllRequests: Promise<void>;
-  private deleteService: Function;
+  private deleteService: Function = () => {};
```

Comment:
`private deleteService!: Function` - the `!` means it will be initialized before any usage.

Reply:
Got it, done.